blazor/diagram/nodes/events.md

```diff
@@ -256,6 +258,52 @@ The following code example explains how to get the selection change event in the
     }
 }
 ```
+## NodeCreating event
+
+* [NodeCreating](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.SfDiagramComponent.html#Syncfusion_Blazor_Diagram_SfDiagramComponent_NodeCreating) helps to define default properties of node. Node creating is triggered  when diagram is initialized. In the Node creating event, we can customize the node properties. 
```

Comment:
* The [NodeCreating](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.Diagram.SfDiagramComponent.html#Syncfusion_Blazor_Diagram_SfDiagramComponent_NodeCreating) helps you to define the default properties of the node. The node creation is triggered when the diagram is initialized. In the node creating event, customize the node properties.